src/wp-includes/wp-db.php

```diff
@@ -1564,7 +1564,17 @@ public function print_error( $str = '' ) {
 			'error_str' => $str,
 		);
 
-		if ( $this->suppress_errors ) {
+		/**
+		 * Filters whether to suppress a query error.
+		 *
+		 * @since 6.0.0
+		 *
+		 * @param boolean $suppress_error Whether to suppress the error.
+		 * @param string  $error_str      Error string.
```

Comment:
These variable names should match the one passed to the filter hook unless there is a good reason. I think the `$query` one makes sense, but can `$str` be changed?